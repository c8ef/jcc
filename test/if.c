int foo(void) {
  if (1) {
    int x = 42;
  } else {
    double y = 42.0;
  }
}
