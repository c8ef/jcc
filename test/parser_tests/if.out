FunctionDecl: foo
  Args[0]
  CompoundStatement
    IfStatement
      IntergerLiteral: 1
      CompoundStatement
        DeclStatement
          VarDecl: x
            IntergerLiteral: 42
      CompoundStatement
        DeclStatement
          VarDecl: y
            IntergerLiteral: 42
